lms/djangoapps/organizations/tests.py

```diff
@@ -611,3 +612,139 @@ def test_organizations_metrics_get_courses_filter(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data['users_grade_complete_count'], 0)
         self.assertEqual(response.data['users_grade_average'], 0)
+
+    def test_organizations_groups_users_get(self):
+        organization = self.setup_test_organization()
+        group = GroupFactory.create()
+        users = UserFactory.create_batch(5)
+        group.organizations.add(organization['id'])
```

Comment:
I think we should add this group to one more organization to test it works as expected when same group is in multiple organizations.